application/web/web.js

```diff
@@ -209,9 +225,12 @@ function openEntryforEdit() {
 
 /* Identical to cancelEntries() for now, more functionality to come. */
 function saveCurrentEntry() {
-    
-    const titleTextArea = document.querySelector('.title-textarea');
+    hideTextEditor();
+
+    const titleTextArea = document.querySelector('#title-input');
+  
```

Comment:
can also be removed/commented

Reply:
I think I need this line to be able to clear the textarea on click